src/theme/texts.ts

```diff
@@ -1,5 +1,5 @@
-export const textStyles = {
-  p: {
-    textTransform: 'uppercase',
-  },
-};
+// export const textStyles = {
```

Comment:
if completely commented this out this file can go

Reply:
There is a bunch that can go! I just didn't want to gut everything on my first PR.